api/catalog/api/constants/field_values.py

```diff
@@ -0,0 +1,31 @@
+AUDIO_CATEGORIES = {
```

Comment:
I think we should segregate the file by media types, or use `AUDIO_`/`IMAGE_` prefix before every variable. Otherwise it'll quickly become confusing as we add more media types.

Reply:
I was thinking about this...but there are some fields that might be relevant across media types. Like `DURATION` - I could see that being used for both audio _and_ video.